action.py

```diff
@@ -135,15 +135,17 @@ def _fatal_help(msg):
 else:
     _summary("❌ pip-audit found one or more problems")
 
-    with open("/tmp/pip-audit-output.txt", "r") as io:
-        output = io.read()
+    p = Path("/tmp/pip-audit-output.txt")
+    if p.exists():
+        with p.open() as io:
+            output = io.read()
 
-        # This is really nasty: our output contains multiple lines,
-        # so we can't naively stuff it into an output.
-        print(f"output={b64encode(output.encode()).decode()}", file=_GITHUB_OUTPUT)
+            # This is really nasty: our output contains multiple lines,
+            # so we can't naively stuff it into an output.
+            print(f"output={b64encode(output.encode()).decode()}", file=_GITHUB_OUTPUT)
 
-        _log(output)
-        _summary(output)
+            _log(output)
+            _summary(output)
```

Comment:
Rather than check-and-skip, let's try-and-fail:

```python
try:
    # open
except OSError as exc:
   # log an explicit error here, before the full stdout log
```